src/components/victory-line/victory-line.js

```diff
@@ -40,12 +43,39 @@ export default class VictoryLine extends React.Component {
   static defaultTransitions = {
     onExit: {
       duration: 500,
-      before: (datum) => ({ y: datum.y })
+      before: (datum) => ({ y: datum.y }),
+      beforeClipPathWidth: (data, child, exitingNodes) => {
```

Comment:
Is this identical to what you are using in VictoryArea? Can you extract this code into a helper. The code for the default could even live in `transition.js` we kept the default `before` / `after` functions here for the others because they were different for each type of component. If `beforeClipPathWidth` etc is always the same you can keep the code with the rest of the transition stuff.